ieducar/scripts/install_pear_packages.sh

```diff
@@ -1,17 +0,0 @@
-# este arquivo deve ser usado para instalar as dependencias do i-educar,
-#
-# para instalar o pear no ubuntu: sudo apt-get install php-pear
-# ou para atualizar uma instalação já existente: sudo pear upgrade pear
-
-echo -e "\n\n** Instalando dependencias i-Educar"
-
-sudo pear install XML_RPC2
-
-# mailer
-sudo pear install Mail
-sudo pear install Net_SMTP
-
-# recaptcha
-sudo pear install Services_ReCaptcha
-
-sudo service apache2 restart
```

Comment:
Arquivo citados em vários outros arquivos:
- ieducar/lib/Portabilis/Mailer.php
- ieducar/lib/Portabilis/Utils/ReCaptcha.php
- ieducar/lib/Portabilis/Report/ReportFactoryRemote.php

Reply:
Farei um ack a respeito das chamadas

Reply:
Notícias... O `ack` não encontrou nenhuma referencia a tal script no projeto

Reply:
Pode deixar apagado, mas pode remover os comentário tbm por favor?

Reply:
Farei ASAP